lib/src/components/NxDropdown/NxDropdown.tsx

```diff
@@ -77,7 +83,10 @@ const _NxDropdown = forwardRef<HTMLDivElement, Props>(function NxDropdown(props,
 _NxDropdown.propTypes = propTypes;
 
 const NxDropdown = Object.assign(_NxDropdown, {
-  Divider: withClass('hr', 'nx-dropdown__divider')
+  Divider: withClass('hr', 'nx-dropdown__divider', 'separator'),
```

Comment:
It's bad practice to re-specify roles that are already the default on a given type of HTML element. `separator` is in fact the default role for `<hr>`.

Reply:
> It's bad practice to re-specify roles that are already the default on a given type of HTML element. `separator` is in fact the default role for `<hr>`.

Ops, yes I removed it here: 103669afeb61fd3dfd3ac5172c4e6d3a1951aa6e